tor/core/validation.py

```diff
@@ -16,34 +26,51 @@ def _header_check(reply, config, tor_link=ToR_link):
 
 
 def _thread_title_check(original_post, history_item):
-    # Verify that the link titles match. On the original post, it will be
-    # removed, but we should still be able to extract the title of the
-    # submission it's on. Then we check to see if the title for that submission
-    # is in the r/ToR post, mirroring the max line truncation that's in
-    # posts.py.
+    """
+    Verify that the link titles match. On the original post, it will be
+    removed, but we should still be able to extract the title of the
+    submission it's on. Then we check to see if the title for that submission
+    is in the r/ToR post, mirroring the max line truncation that's in
+    posts.py.
+
+    :param original_post: Comment object; comment that says "done".
+    :param history_item: Comment object; comment pulled from user's history.
+    """
     max_title_length = 250
     return (
         history_item.link_title[:max_title_length - 4] in
         original_post.link_title
     )
 
 
+def _thread_author_check(original_post, history_item, config):
+    return (
```

Comment:
Not required, but a docstring/comment here to warn that this will hit the API instead of using locally cached data might be nice. This method is short enough that it should be relatively apparent, but I want to make sure we're policing our API usage pretty strictly.

Reply:
Oh yeah, I'll change that real quick and then merge.